rpa-builder/6.4/modules/ROOT/pages/_partials/aws-doc-processing.adoc

```diff
@@ -0,0 +1,7 @@
+// tag::general[]
+Action steps that use Amazon Textract services process the document in a synchronous fashion, therefore the document is not stored by Amazon. These action steps automatically convert PDF files to PNG images. You can adjust the fidelity of the conversion in the properties grid of the action step. 
+// end::general[]
+
+// tag::specific[]
+This action step processes the document in a synchronous fashion, therefore the document is not stored by Amazon, and automatically convert PDF files to PNG images. You can adjust the fidelity of the conversion in the properties grid. 
```

Comment:
Sorry to say I can't get the meaning of this sentence. Do you mean?:

- This action step processes the document in a synchronous fashion, therefore Amazon does not store the document, and it automatically converts PDF files to PNG images. 
or
- This action step processes the document in a synchronous fashion, therefore Amazon does not store the document. These action steps automatically convert PDF files to PNG images.

Reply:
Yes! This version read well! 👍